### ISSUE TITLE
KeyError when calling Release()

### ISSUE THREAD

**Issue by dhodovsk** [red-hat-storage/errata-tool#166]
This error is shown in our automation for every try to update errata. Could youplease investigate?
```
>>> from errata_tool.release import Release
>>> Release(name='Extras-RHEL-7.6')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/usr/local/lib/python3.7/site-packages/errata_tool/release.py", line 29, in __init__
    self.refresh()
  File "/usr/local/lib/python3.7/site-packages/errata_tool/release.py", line 52, in refresh
    self.is_pdc = self.data['attributes']['is_pdc']
KeyError: 'is_pdc'
```

**Comment by simonbaird** [comment]
We removed a lot of PDC code in ET 3.20.0 which went live a few hours ago, and didn't consider the API change. I expect that's the reason for this missing attribute. See [BZ#1692965](https://bugzilla.redhat.com/show_bug.cgi?id=1692965).

**Comment by ktdreyer** [comment]
I'll work on fixing this today.

**Comment by ktdreyer** [comment]
When I refresh the static JSON fixtures in the tree based on today's live errata.devel.redhat.com responses, the test suite reproduces this crash.

**Comment by ktdreyer** [comment]
@dhodovsk would you please test the patch at https://github.com/red-hat-storage/errata-tool/pull/167 and report if this works for you?

**Comment by ktdreyer** [comment]
I've just released v1.20.0 today with this change. If it resolves this issue for you, would you please close this GitHub ticket?
